apps/functions-subscription/api/internal.yaml

```diff
@@ -267,6 +267,35 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
+  subscriptions/{trialId}/{userId}:
```

Comment:
**issue(blocking)**: follow other endpoints specification

why are you creating a new endpoint? 
if we are talking about rest, the subscription resource exists only if the trial resource exists, that's why other endpoints are in the format `/trials/{trialId}/subscriptions`.

I think you can re-use [the endpoint already defined](https://github.com/pagopa/trial-system/blob/6a6b6f98018747e33b564c75178505f4b251f8cf/apps/functions-subscription/api/internal.yaml#L30-L31)

or do you have any issues that make you define a new endpoint?